packages/fxa-content-server/tests/functional/security_events.js

```diff
@@ -0,0 +1,143 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { registerSuite } = intern.getInterface('object');
+const TestHelpers = require('../lib/helpers');
+const FunctionalHelpers = require('./lib/helpers');
+const selectors = require('./lib/selectors');
+
+const {
+  clearBrowserState,
+  click,
+  createUser,
+  fillOutResetPassword,
+  fillOutSignIn,
+  noSuchElement,
+  openPage,
+  testElementExists,
+  testElementTextInclude,
+  testElementValueEquals,
+  type,
+} = FunctionalHelpers;
+
+const config = intern._config;
+const SIGNIN_PAGE_URL = `${config.fxaContentRoot}signin`;
+const SECURITY_EVENTS_URL = `${config.fxaContentRoot}security_events`;
+const PASSWORD = '12345678';
+let email;
+
+registerSuite('security_events', {
+  beforeEach: function() {
+    email = TestHelpers.createEmail();
+
+    return this.remote
+      .then(createUser(email, PASSWORD))
```

Comment:
This should set the `preverified` flag to true to create a verified account, ref https://github.com/mozilla/fxa/blob/8701348cdd79dbdc9879b2b4a55a23a135a32bc1/packages/fxa-content-server/tests/functional/sign_in_blocked.js#L39.